docs/assets/css/main.css

```diff
@@ -10459,6 +10459,10 @@ button.bg-dark:focus {
 
 .text-center {
     text-align: center!important;
+<<<<<<< HEAD
```

Comment:
@raghavaggarwal99 we may have some left over merge conflict annotations here.

Reply:
Ya i actually having some git issues.

Reply:
RIght now i am creating the pull request, will see these git issues a bit later